Helper/Invoice.php

```diff
@@ -170,13 +170,8 @@ public function createInvoice(Order $order, Notification $notification, bool $is
                 // set transaction id so you can do a online refund from credit memo
                 $invoice->setTransactionId($notification->getPspreference());
 
-                $createPendingInvoice = (bool)$this->configHelper->getConfigData(
-                    'create_pending_invoice',
-                    'adyen_abstract',
-                    $order->getStoreId()
-                );
 
-                if ((!$isAutoCapture) && ($createPendingInvoice)) {
```

Comment:
Could you please clarify in the description why this change is required?

Reply:
Hello,
There is no configuration key with code create_pending_invoice on the module.
The variable $createPendingInvoice  is null on all the case so the if ((!$isAutoCapture) && ($createPendingInvoice)) will return false every time.
I think we should only control on the variable if (!$isAutoCapture) .

Reply:
Hello @Marwaa92,

Thank you for your detailed explanation and contribution. We will merge this PR and release.

Best Regards,
Can